spec/spec.js

```diff
@@ -0,0 +1,96 @@
+describe('type-of-is', function(){
```

Comment:
this doesn't improve test coverage it just changes the syntax. not interested in including this.